modules/gapi/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.3)
```

Comment:
Why is 3.3?

OpenCV requires 3.5.1+ itself. Lower versions are not tested at all (however they may work).

Reply:
IE uses G-API standalone build for its stuff but required version of cmake is 3.3 in IE. So, I think it's reasonable to put lower version in here